forge-gui/res/cardsfolder/upcoming/vitu_ghazi_inspector.txt

```diff
@@ -0,0 +1,13 @@
+Name:Vitu-Ghazi Inspector
+ManaCost:1 G
+Types:Creature Elf Detective
+PT:1/3
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. (Exile cards with total mana value 6 or greater from your graveyard.)
+K:Reach
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | CheckSVar$ Collected | ValidCard$ Card.Self | Execute$ TrigPutCounter | TriggerDescription$ When CARDNAME enters the battlefield, if evidence was collected, put a +1/+1 counter on target creature and you gain 2 life.
```

Comment:
did you check the Condition works?

Reply:
Hmm it's not working. I don't think I'm understanding how the collected condition works ....

Reply:
I've used the same method like you showed below. It's working now

Reply:
no, that's not how the card I've mentioned does it

Reply:
`Hmm ConditionDefined$ Collected | ConditionPresent$ Card  isn't working `

Reply:
well the script has `ConditionDefined$ CastSA>Collected` so how about using that 🦀

Reply:
Yeah I tried that first but that didn't work